python/3.11/Dockerfile

```diff
@@ -4,11 +4,12 @@ FROM $BUILD_FROM
 ARG \
     PYTHON_VERSION \
     PIP_VERSION \
-    GPG_KEY \
+    CERT_IDENTITY \
+    CERT_OIDC_ISSUER \
     QEMU_CPU
 
 # ensure local python is preferred over distribution python
-ENV PATH /usr/local/bin:$PATH
+ENV PATH=/usr/local/bin:$PATH
```

Comment:
Hm, I think the space variant also did not expand $PATH... I wonder how certain things even worked with that in mind :thinking: 

Anyways, this is the right thing to do :+1: